ecommerce/managing_products/variants.rst

```diff
@@ -1,92 +1,115 @@
+Manage product variants
+=======================
 
-Product variants are used to offer variations of the same product to your
-customers on the products page. For example, the customer chooses a T-shirt
-and then selects its size and color.  In the example below, the customer
-chooses a phone, and then selects the memory; color and Wi-Fi band from the
-available options.
+Product variants are used to offer variations of the same product to your customers on the
+product's page.
 
-.. image:: ./media/variants.png
-   :align: center
+Let's say, one of your customer selects a T-shirt from your product catalog.
+
+Then, they choose the size and color they want. The various size and color combinations are known
+as product variants.
+
+Create attributes & variants
+============================
+
+In order to create various attributes and variants for your products, you first need to turn on
+*Variants* in :menuselection:`Sales --> Configuration --> Settings`.
+
+Then, select a product from the Products list (:menuselection:`Products --> Products`). When the
+product detail form opens, click *Edit*, and go to the *Variants* tab.
+
+Here, you can add as many different attributes as you'd like, by clicking *Add a line*.
 
-How to create attributes & variants
-===================================
+These attributes appear as 3 different types:
 
-* Turn on *Products can have several attributes, defining variants (Example:
-  size, color,...)* in :menuselection:`Sales --> Settings`.
+* Radio buttons
+* Color buttons
+* Drop-down menu
 
-* Select a product from the Products list, go to the *Variants* tab.
+Several variants appear as soon as there are 2 values for 1 attribute.
 
-* Add as many attributes as you need from 3 different types: radio buttons,
-  drop-down menu or color buttons. You get several variants as soon as there are
-  2 values for 1 attribute.
+Don't forget to hit *Save* once all variants have been entered.
 
-.. image:: ./media/variants_setup.png
+.. image:: ./variants/variants-values.png
    :align: center
+   :alt: ecommerce product variants values
 
-How to edit variants
-====================
+Edit variants
+=============
 
-See all the variants from the product template detail form.
+After you hit *Save*, you are able to see all the variants from the product template detail form,
+via the *Variants* smart button.
 
-.. image:: ./media/variants_button.png
+.. image:: ./variants/variants-smart-button.png
    :align: center
-   :scale: 50
-   :width: 50
-   :height: 50
+   :alt: ecommerce product variants smart button
 
-You can edit following data:
+Clicking the *Variants* smart button takes you to the Product Variants page, where you can edit
+following data for each specific variant:
 
-* Picture (will update in real time on the website),
-* Barcode,
-* Internal Reference (SKU #),
-* Volume,
-* Weight,
-* Active (available in quotes & website).
+* Picture(s)
+* Internal Reference (SKU #)
+* Barcode
+* Volume
+* Weight
+* Cost
```

Comment:
* Changing the display style (trailing comma versus no) just adds noise to the diff for no value.
* So does reordering lines even when they don't change (Barcode and SKU being swapped).
* If multiple pictures *can* be associated with a variant, then strict plural seems fine, and is simpler.